src/commands/dev/lib/dev-builder.ts

```diff
@@ -203,154 +179,36 @@ export async function executeBuild(
-          };
-          if (typeof builder.subscribe === 'function') {
-            // If the builder has a `subscribe()` function, then the initial
-            // builds are not run and instead an array of "subscriptions" is
-            // created. When an HTTP request comes in that matches one of the
-            // subscription patterns, then this builder will be actually be
-            // executed.
-            devServer.output.debug(`Getting subscriptions for ${entry.fsPath}`);
-            const subscription = await builder.subscribe(buildConfig);
-            devServer.subscriptions.push({
-              type: 'Subscription',
-              patterns: subscription,
-              buildConfig: build,
-              buildEntry: entry,
-              buildTimestamp: Date.now()
-            });
-            continue;
-          } else {
-            devServer.output.debug(
-              `Building ${entry.fsPath} with "${build.use}" v${
-                builder[PACKAGE]!.version
-              } (workPath = ${workPath})`
-            );
-            output = await builder.build({ ...buildConfig, workPath });
-
-            if (typeof builder.prepareCache === 'function') {
-              const cachePath = getWorkPath();
-              await mkdirp(cachePath);
-              build.builderCachePromise = executePrepareCache(
-                devServer,
-                build,
-                {
-                  files,
-                  entrypoint,
-                  workPath,
-                  cachePath,
-                  config,
-                  meta: { isDev: true, requestPath: null }
-                }
-              );
-            }
-          }
-        } finally {
-          devServer.restoreOriginalEnv();
-        }
-
-        // enforce the lambda zip size soft watermark
-        const builderConfig = builder.config || {};
-        const { maxLambdaSize = '5mb' } = { ...builderConfig, ...config };
-        let maxLambdaBytes: number;
-        if (typeof maxLambdaSize === 'string') {
-          maxLambdaBytes = bytes(maxLambdaSize);
-        } else {
-          maxLambdaBytes = maxLambdaSize;
-        }
-
-        for (const asset of Object.values(output)) {
-          if (asset.type === 'Lambda') {
-            const size = asset.zipBuffer.length;
-            if (size > maxLambdaBytes) {
-              throw new LambdaSizeExceededError(size, maxLambdaBytes);
-            }
-          }
-
-          asset.buildConfig = build;
-          asset.buildEntry = entry;
-        }
+  cwd: string
+): Promise<BuildMatch[]> {
+  const matches: BuildMatch[] = [];
+  const builds = nowJson.builds || [{ src: '**', use: '@now/static' }];
+  for (const buildConfig of builds) {
+    let { src, use } = buildConfig;
+    if (src[0] === '/') {
+      // Remove a leading slash so that the globbing is relative to `cwd`
+      // instead of the root of the filesystem. This matches the behavior
+      // of Now deployments.
```

Comment:
Maybe we should sanitize the leading `/` at the beginning when we read `buildConfig` from `now.json`. Seems no reason to preserve that.

Reply:
Hmmm, perhaps. Maybe let's worry about that once we have some `now dev` tests in place.